seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/main/java/org/apache/seata/spring/boot/autoconfigure/properties/SeataProperties.java

```diff
@@ -51,6 +51,12 @@ public class SeataProperties {
      * Whether use JDK proxy instead of CGLIB proxy
      */
     private boolean useJdkProxy = false;
+    /**
+     * Whether to expose the proxy object through AopContext.
+     * Setting this to true allows AopContext.currentProxy() to be used to obtain the current proxy,
+     * which can be useful for invoking methods annotated with @GlobalTransactional within the same class.
+     */
+    private boolean exposeProxy = false;
```

Comment:
https://github.com/apache/incubator-seata/tree/2.x/script/client/spring
Please add the configuration example here